app/src/main/java/de/vectordata/skynet/net/PacketHandler.java

```diff
@@ -104,6 +101,11 @@ void handlePacket(byte id, byte[] payload) {
             if (flags != null)
                 if ((message.messageFlags | flags.value()) != message.messageFlags)
                     throw new IllegalStateException(String.format("Incoming channel message lacks required message flags (got: %s, required at least: %d)", message.messageFlags, flags.value()));
+
+            if (message.isCorrupted) {
```

Comment:
Users should not get annoying dialog windows if someone else is using a malicious client. Marking the respective chat message as corrupted is enough.

```suggestion
            if (message.isCorrupted && ((message.messageFlags & MessageFlags.LOOPBACK) == 1)) {
```

Reply:
So users should not be notified after sync if the client receives corrupted chat messages?